### DISCUSSION TITLE
Is Kernel Stack Protection disabled for Sandboxed programs?

### DISCUSSION THREAD

**Discussion by ghost** [sandboxie-plus/Sandboxie#3792] [Q&A Feedback]
Template.ini says:

> _CallEnclave causes a BSOD when kernel stack protection is enabled_
> ```
> DisableWinNtHook=CreateEnclave
> DisableWinNtHook=LoadEnclaveData
> DisableWinNtHook=InitializeEnclave
> DisableWinNtHook=CallEnclave
> DisableWinNtHook=TerminateEnclave
> ```

Does that mean that while Sandboxie does not allow Kernel Stack Protection to function inside sandboxes? Sandboxie executable image itself does have Kernel Stack Protection (CET) flag.

**Comment by DavidXanatos** [comment]
No we don't have the ability to disable Stack Protection in the kernel, hence we disable the hooks for the enclave functions.

**Comment by ghost** [comment]
Off-topic, but what would you recommend for hardware-level isolation? Windows Sandbox?
